.github/workflows/nightly.yml

```diff
@@ -104,4 +104,5 @@ jobs:
       run: |
         docker login -u ${{ secrets.QUAY_USER }} -p ${{ secrets.QUAY_PASSWORD }} quay.io
 
-        make -e DOCKER_CLI_EXPERIMENTAL=enabled build-linux-multi-arch container-multi-arch-push-kiali-quay
\ No newline at end of file
+        make -e DOCKER_CLI_EXPERIMENTAL=enabled build-linux-multi-arch container-multi-arch-all-push-kiali-quay
+        
```

Comment:
While we are here, please remove this line with the spaces :) It is trivial, but lines with spaces in them bother me for some reason :) 
```suggestion
```